src/model/transaction/TransferTransaction.ts

```diff
@@ -88,6 +89,21 @@ export class TransferTransaction extends Transaction {
         super(TransactionType.TRANSFER, networkType, version, deadline, fee, signature, signer, transactionInfo);
     }
 
+    /**
+     * Return the string notation for the set recipient
+     * @internal
+     * @returns {string}
+     */
+    protected recipientToString(): string {
```

Comment:
test on this new method?

Reply:
added in [#00938be](https://github.com/nemtech/nem2-sdk-typescript-javascript/pull/83/commits/00938bee4f125e7e0b10aedb3cc5ab4aea8fd7e8) :+1: